_includes/post-meta.html

```diff
@@ -0,0 +1,7 @@
+{% if post.author %}
+  <span class="post-date site-subheading">By {{ post.author }}</span>
```

Comment:
Do we need the `post-date` class for the author?

```suggestion
  <span class="site-subheading">By {{ post.author }}</span>
```

---

Also, we should double-check, on screen readers how all caps affect how the subheading is read.

Reply:
I don't hear anything related to text-transform in VO. @amycole501 and @amycole501 can you let us know if your screen readers differentiate between casing types?

Reply:
Another note: updated `post-date` to `post-meta` in https://github.com/uswds/uswds-site/commit/41fd023648d7797dcb9494cc5f86b21c8871495a